doc/api/tracing.md

```diff
@@ -48,4 +48,6 @@ as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
 
-[Performance API]: perf_hooks.html
+[async_hooks]: async_hooks.md
```

Comment:
As `.md` files are only sources for `.html` documents in https://nodejs.org/api/, we link to HTML docs. Also, we prefer ASCII sorting in bottom references, So this fragment may be written in such way:
```md
[Performance API]: perf_hooks.html
[V8]: v8.html
[async_hooks]: async_hooks.html
```

Reply:
@vsemozhetbyt, learned the documentation scripts (i.e. `make doc`) & the built `.html` files' relationship to the docs. Will keep that in mind going forward. 🙏for this piece of valuable review comment.